Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/EditorComponentModeBus.h

```diff
@@ -142,6 +142,10 @@ namespace AzToolsFramework
             /// has the same Component on it, move all Components into ComponentMode.
             virtual void AddSelectedComponentModesOfType(const AZ::Uuid& componentType) = 0;
 
+            // If the user changes component mode through the switcher, switch the 
+            // component mode seamlessly.
```

Comment:
suggestion: This section of code doesn't really know anything about the switcher, so referencing it here might not be a good idea. Just saying `Switches the Component Mode immediately` (as this should work from somewhere else that might not want to use the switcher in future)